<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrencyHistoryDialog</class>
 <widget class="QWidget" name="CurrencyHistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Currency History</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Currency History: USD - United States Dollar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="versionListWidget">
      <property name="minimumHeight">
       <number>150</number>
      </property>
      <property name="maximumHeight">
       <number>250</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Version</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modified At</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modified By</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="detailWidget" native="true">
      <layout class="QVBoxLayout" name="detailLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="changesTab">
          <attribute name="title">
           <string>Changes</string>
          </attribute>
          <layout class="QVBoxLayout" name="changesTabLayout">
           <item>
            <widget class="QTableWidget" name="changesTableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>Field</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Old Value</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Value</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="fullDetailsTab">
          <attribute name="title">
           <string>Full Details</string>
          </attribute>
          <layout class="QVBoxLayout" name="fullDetailsTabLayout">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>560</width>
                <height>450</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="isoCodeLabel">
                 <property name="text">
                  <string>ISO Code:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="isoCodeValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="nameLabel">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="nameValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="numericCodeLabel">
                 <property name="text">
                  <string>Numeric Code:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="numericCodeValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="symbolLabel">
                 <property name="text">
                  <string>Symbol:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="symbolValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="fractionSymbolLabel">
                 <property name="text">
                  <string>Fraction Symbol:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="fractionSymbolValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="fractionsPerUnitLabel">
                 <property name="text">
                  <string>Fractions Per Unit:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLabel" name="fractionsPerUnitValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="versionNumberLabel">
                 <property name="text">
                  <string>Version:</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLabel" name="versionNumberValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="modifiedByLabel">
                 <property name="text">
                  <string>Modified By:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLabel" name="modifiedByValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="modifiedAtLabel">
                 <property name="text">
                  <string>Modified At:</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QLabel" name="modifiedAtValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
